conans/client/conf/required_version.py

```diff
@@ -1,9 +1,16 @@
 from conans.client.cache.cache import ClientCache
-from semver import satisfies, Range
+from semver import satisfies
 from conans import __version__ as client_version
 from conans.errors import ConanException
 
 
+def validate_conan_version(required_range):
+    result = satisfies(client_version, required_range, loose=True)
+    if not result:
+        raise ConanException("The current Conan version ({}) does not match to the required"
```

Comment:
```suggestion
        raise ConanException("The current Conan version ({}) does not match the recipe required"
```

Reply:
I see this message is used to raise from Conan required version from config as well as from recipe, but would like a more clear message about it

Reply:
Cannot be changed, because it is common.

Reply:
I have improved the message grammar, and added a capture of the error, so when you are reading a conanfile of a dependency it will throw and print:

```bash
pkg/version@user/channel: ERROR: Cannot load recipe
The conanfile at path .... version not satisfied message
```